code/obj/decal/cleanable.dm

```diff
@@ -619,7 +619,7 @@ var/list/blood_decal_violent_icon_states = list("floor1", "floor2", "floor3", "f
 	desc = "You can try to clean it up, but there'll always be a little bit left."
 	icon = 'icons/effects/glitter.dmi'
 	icon_state = "glitter"
-	random_dir = EAST
+	random_dir = 1
```

Comment:
I have bad news, these numbers meant something, look at `decal.dm:13`. Really though, it should just be a pair of defines like `RANDOM_DIR_ALL` and `RANDOM_DIR_CARDINAL`